conf/messages

```diff
@@ -1,2 +1,77 @@
 service.name = si-protected-user-list-admin-frontend
 service.text = This is your new service
+home.page.title = MDTP User Admin - Silent Login
```

Comment:
Pls can you change all strings in this file from "Silent Login" to "SI Protected User List", thx

Reply:
done